### PR TITLE
squid: bump to 3.5.27

### PR DESCRIPTION
**Pull request by ratkaj**
Maintainer: Marko Ratkaj <marko.ratkaj@sartura.hr> / @ratkaj
Compile tested: LEDE master, mvebu
Run tested: LEDE master, mvebu, Linksys wrt3200

Approved by Luka Perkov @lperkov 

Description:
Bump Squid to version 3.5.27 and fix broken menuconfig menu.

Signed-off-by: Marko Ratkaj <marko.ratkaj@sartura.hr>

### CODE REVIEW

File: net/squid/Makefile [hunk 1]
```diff
@@ -8,15 +8,15 @@
...
 PKG_NAME:=squid
-PKG_VERSION:=3.5.12
+PKG_VERSION:=3.5.27
 PKG_RELEASE:=1
 
 PKG_LICENSE:=GPL-2.0
 PKG_MAINTAINER:=Luka Perkov <luka@openwrt.org>
 
 PKG_SOURCE:=$(PKG_NAME)-$(PKG_VERSION).tar.xz
 PKG_SOURCE_URL:=http://www.squid-cache.org/Versions/v3/3.5/
```
Comment (diizzyy) [inline net/squid/Makefile]: You can add mirrors first and fallback to mainsite?
```
http://www2.pl.squid-cache.org/
http://www3.us.squid-cache.org/
http://www.squid-cache.org/
```

**Comment by diizzyy** [comment]
Why are we patching the mime.conf file?

**Comment by ratkaj** [comment]
That is a very good question. Mime patch exists since squid was imported from oldpackages and i cant find a reference in git history. Ill test if it is really needed before i update this pull request

**Comment by ratkaj** [comment]
Removed mime patch and added mirrors

**Comment by champtar** [comment]
Hi @ratkaj 
If you are the new maintainer please update `PKG_MAINTAINER`
in any case @lperkov please ack

**Comment by lperkov** [comment]
ACK

**Comment by ratkaj** [comment]
PKG_MAINTAINER updated

**Pull request merged**